scripts/ci/ci_pylint.sh

```diff
@@ -20,11 +20,29 @@
 # Script to run Pylint on all code. Should be started from root directory of Airflow:
 # ./scripts/ci/ci_pylint.sh
 
-set -ex
+set -x
 
 find . -name "*.py" \
 -not -path "./.eggs/*" \
 -not -path "./airflow/www/node_modules/*" \
 -not -path "./airflow/_vendor/*" \
 -not -path "./build/*" \
+-not -path "./tests/*" \
 -not -name 'webserver_config.py' | grep -vFf scripts/ci/pylint_todo.txt | xargs pylint --output-format=colorized
+RES_MAIN=$?
+
+find . -name "*.py" \
+-path "./tests/*" \
```

Comment:
Could remove the `-path` condition:

```
find tests/ -name "*.py" \
```

Reply:
Could not do it because then exclusion will not work properly. I will stay with -path